examples/stack/bqc/example_bqc_nv.py

```diff
@@ -0,0 +1,253 @@
+        self, context: ProgramContext
+    ) -> Generator[EventExpression, None, Dict[str, Any]]:
+        conn = context.connection
+        epr_socket = context.epr_sockets[self.PEER]
+        csocket: ClassicalSocket = context.csockets[self.PEER]
+
+        # Create EPR Pair
+        epr1, epr2 = epr_socket.recv_keep(2)
+        epr2.cphase(epr1)
+
+        yield from conn.flush()
+
+        delta1 = yield from csocket.recv_float()
+
+        epr2.rot_Z(angle=delta1)
+        epr2.H()
+        m1 = epr2.measure(store_array=False)
+        yield from conn.flush()
+
+        m1 = int(m1)
+
+        csocket.send_int(m1)
+
+        delta2 = yield from csocket.recv_float()
+
+        epr1.rot_Z(angle=delta2)
+        epr1.H()
+        conn.insert_breakpoint(BreakpointAction.DUMP_LOCAL_STATE)
+        m2 = epr1.measure(store_array=False)
+        yield from conn.flush()
+
+        m2 = int(m2)
+        return {"m1": m1, "m2": m2}
+
+
+PI = math.pi
+PI_OVER_2 = math.pi / 2
+
+
+def computation_round(
+    cfg: StackNetworkConfig,
+    num_times: int = 1,
+    alpha: float = 0.0,
+    beta: float = 0.0,
+    theta1: float = 0.0,
+    theta2: float = 0.0,
+) -> None:
+    client_program = ClientProgram(
+        alpha=alpha,
+        beta=beta,
+        trap=False,
+        dummy=-1,
+        theta1=theta1,
+        theta2=theta2,
+        r1=0,
+        r2=0,
+    )
+    server_program = ServerProgram()
+
+    _, server_results = run(
+        cfg, {"client": client_program, "server": server_program}, num_times=num_times
+    )
+
+    m2s = [result["m2"] for result in server_results]
+    num_zeros = len([m for m in m2s if m == 0])
+    frac0 = round(num_zeros / num_times, 2)
+    frac1 = 1 - frac0
+    print(f"dist (0, 1) = ({frac0}, {frac1})")
+
+
+def trap_round(
+    cfg: StackNetworkConfig,
+    num_times: int = 1,
+    alpha: float = 0.0,
+    beta: float = 0.0,
+    theta1: float = 0.0,
+    theta2: float = 0.0,
+    dummy: int = 1,
+) -> None:
+    client_program = ClientProgram(
+        alpha=alpha,
+        beta=beta,
+        trap=True,
+        dummy=dummy,
+        theta1=theta1,
+        theta2=theta2,
+        r1=0,
+        r2=0,
+    )
+    server_program = ServerProgram()
+
+    client_results, server_results = run(
+        cfg, {"client": client_program, "server": server_program}, num_times=num_times
+    )
+
+    p1s = [result["p1"] for result in client_results]
+    p2s = [result["p2"] for result in client_results]
+    m1s = [result["m1"] for result in server_results]
+    m2s = [result["m2"] for result in server_results]
+
+    assert dummy in [1, 2]
+    if dummy == 1:
+        num_fails = len([(p, m) for (p, m) in zip(p1s, m2s) if p != m])
+    else:
+        num_fails = len([(p, m) for (p, m) in zip(p2s, m1s) if p != m])
+
+    frac_fail = round(num_fails / num_times, 2)
+    print(f"fail rate: {frac_fail}")
+
+
+if __name__ == "__main__":
+    num_times = 100
+    LogManager.set_log_level("WARNING")
+
+    # num_times = 1
```

Comment:
I think that this commented code is not needed anymore and that it could be removed?